APP/controller.js

```diff
@@ -66,3 +67,15 @@ exports.postCommentByArticleID = (req, res, next) => {
       next(err);
     });
 };
+
+
+exports.deleteCommentByCommentID = (req, res, next) => {
+  const {comment_id} = req.params
+  removeCommentByCommentID(comment_id)
+  .then((comment) => {
+    res.status(204).send({comment})
+  })
```

Comment:
There's no requirement to sent the comment here, as express doesn't send one. You can simply amend this to be .send() or just simply send the status 204 👍